tests/test_schema.py

```diff
@@ -1200,11 +1200,33 @@ def test_promote_float_to_double() -> None:
     assert isinstance(applied.fields[0].field_type, DoubleType)
 
 
+def test_promote_long_to_int() -> None:
```

Comment:
nit, I dont see a test case for int to long. Can we add that here? 

Theres a corresponding test case for double to float and float to double

Reply:
Good one, added those as well 👍